src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,172 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+/**
+ * Represents a parser for user input given to Duke chatbot.
+ *
+ * @author Conrad
+ */
+public class Parser {
+
```

Comment:
Perhaps we should make `Parser` abstract, since it only contains static methods?